src/main/java/org/breedinginsight/daos/TraitDAO.java

```diff
@@ -181,6 +202,130 @@ public TraitDAO(Configuration config, DSLContext dsl) {
         return Optional.of(parseTraitRecord(record, createdByUser, updatedByUser));
     }
 
+    public List<Trait> createTraitsBrAPI(List<Trait> traits, User actingUser, Program program){
+
+        //TODO: Pass ontology reference
```

Comment:
Do we have a card for creating the ontology and using the reference?

Reply:
https://trello.com/c/4V5bXNBb/166-ont-59-create-brapi-ontology-on-program-create